pkg/plugin/manifest/manifest.go

```diff
@@ -50,6 +50,7 @@ type SonobuoyConfig struct {
 	// to the plugin source would be kept.
 	SourceURL string `json:"source-url,omitempty"`
 
+	Order int `json:"order"`
```

Comment:
Definitely add `,omitempty` on this. That will prevent all the 0 values from having to be explicitly written. Not a problem if they do, but it ensures that the tests work as-is (e.g. wont break existing users) and also cuts down on LOC changes for the goldenfiles.